nomenclature/definitions/variable/industry_types.yaml

```
@@ -0,0 +1,65 @@
+# This list of industries will replace any occurrence of `<industry>`
+# in a variable-definition yaml file
+
+
+#   i.e., The following standard generical structure in economic variables:
+#   CapitalPriceIndex|<industry>:
+#   LaborPriceIndex|<industry>:
+
+<Industry>:
+  iAGRI:
+    description: 'Agriculture'
+
+  iCOAL:
+    description: 'Coal extraction'
+
+  iCOIL:
+    description: 'crude oil and natural gas extraction'
```

minor consistency update
```suggestion
    description: 'Crude oil and natural gas extraction'
```